docs/collectors/MonitCollector.md

```diff
@@ -21,6 +21,8 @@ measure_collector_time | False | Collect the collector run time in ms | bool
 metrics_blacklist | None | Regex to match metrics to block. Mutually exclusive with metrics_whitelist | NoneType
 metrics_whitelist | None | Regex to match metrics to transmit. Mutually exclusive with metrics_blacklist | NoneType
 send_totals | False | Send cpu and memory totals | bool
+scheme | http | Select scheme http or https | str
```

Comment:
alphabetize

Reply:
The descriptions need to be added to the get_default_config_help method in the collector file since this doc is actually generated from that.